src/index.js

```diff
@@ -1,4 +1,20 @@
-const date = new Date()
-const todayDate = date.toLocaleDateString('de-de');
-document.getElementById("todayDate").innerHTML = todayDate;
+const date = new Date();
+const todayDate = date.toLocaleDateString();
+document.getElementById('todayDate').innerHTML = todayDate;
 console.log(todayDate);
+
+const userInput = document.getElementById('userInput');
+const addButton = document.getElementById('addButton');
+const todoList = document.getElementById('todoList');
+
+addButton.addEventListener(click, addItem);
```

Comment:
This event should be in the form, on submit: https://developer.mozilla.org/en-US/docs/Web/API/HTMLFormElement/submit_event